core/math/a_star_grid_2d.cpp

```diff
@@ -455,7 +455,7 @@ bool AStarGrid2D::_solve(Point *p_begin_point, Point *p_end_point) {
 	bool found_route = false;
 
 	LocalVector<Point *> open_list;
-	SortArray<Point *, SortPoints> sorter;
+	SortArray<Point*, SortPoints> sorter{};
```

Comment:
This is not correct, the space should not have been removed. As for the `{}`, I don't know.

There are many more instances like this in this PR.